exs/mundo_1/ruby/025.rb

```diff
@@ -0,0 +1,8 @@
+print"Digite seu nome: "
```

Comment:
Faltou informar o enunciado do exercício 025

```suggestion
=begin
Desafio 025

Problema: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA" no nome.

Resolução do problema:
=end

print"Digite seu nome: "
```